nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/TestFetchFile.java

```diff
@@ -48,6 +48,19 @@ public void prepDestDirectory() throws IOException {
     }
 
     @Test
+    public void notFound() throws IOException {
+        final File sourceFile = new File("notFound");
```

Comment:
I assume there's no way (besides Mocks and moving the Files method calls out into instance methods) to test for an unreachable file? This unit test tests the "existing" code path, but not the unreachable one.